### PR TITLE
[Merged by Bors] - feat: port Topology.Algebra.Equicontinuity

### PR DESCRIPTION
**Pull request by Felix-Weilacher**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Equicontinuity.lean [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+open Function
+
+open UniformConvergence
+
+@[to_additive]
+theorem equicontinuous_of_equicontinuousAt_one {ι G M hom : Type _} [TSG : TopologicalSpace G]
+    [UM : UniformSpace M] [Group G] [Group M] [TopologicalGroup G] [UniformGroup M]
+    [MonoidHomClass hom G M] (F : ι → hom) (hf : EquicontinuousAt (CoeFun ∘ F) (1 : G)) :
+    Equicontinuous (α := M) (X := G) (CoeFun ∘ F) := by
```
Comment (jakelev) [inline Mathlib/Topology/Algebra/Equicontinuity.lean:26]: Originally this line was just `Equicontinuous (CoeFun ∘ F) := by`

**Comment by jcommelin** [comment]
@jakelev Did you mean to force push? That overwrote all the changes that @Felix-Weilacher has made.

**Comment by jakelev** [comment]
No, I didn't, I didn't realize what that command did... I was trying only to catch my branch up to the original one.

**Comment by Felix-Weilacher** [comment]
I force pushed back to my commit and incorporated some of Mortiz's improvements

File: Mathlib/Topology/Algebra/Equicontinuity.lean [hunk 2]
```diff
@@ -14,6 +14,7 @@ import Mathlib.Topology.Algebra.UniformConvergence
 # Algebra-related equicontinuity criterions
 -/
 
+set_option autoImplicit false -- **TODO** delete this later
```
Comment (jcommelin) [inline Mathlib/Topology/Algebra/Equicontinuity.lean:17]:
```suggestion
```

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**